modules/ROOT/pages/custom-policy-4-reference.adoc

```diff
@@ -478,10 +475,10 @@ information is using a DataWeave expression:
 #[authentication.properties.someProperty]
 ----
 
-Right now, the only way to setting this object, is through the usage of an SDK Mule extension. How to do it, can be found here:
+Right now, the only way to set this object is through the use of a SDK Mule extension. How to do it, can be found here:
 https://docs.mulesoft.com/mule-sdk/v/1.1/authentication-handler
```

Comment:
Since you are modifying this entry, can I ask you to update this link reference?
> https://docs.mulesoft.com/mule-sdk/v/1.1/authentication-handler 

Should be:
> xref:mule-sdk::authentication-handler.adoc